### ISSUE TITLE
After having updated to 5.1 version of kamailio ICSCF doesn't send the UAR towards HSS

### ISSUE THREAD

**Issue by vhuertas** [kamailio/kamailio#1065]
### Description
Hello,

I have just upgraded our IMS core to the very last version. However, when I try to register one user (Alice) the SIP REGISTER does not reach the SCSCF. After having captured packets flow I realized that it was fault of I-CSCF. I saw DWR/DWA messages between HSS and ICSCF so I assume that the Diameter connection is successfully estalbished. However, no UAR message is sent from I-CSCF to HSS.

#### Debugging Data

```
(paste your debugging data here)
```

#### Log Messages
Apr  1 19:45:30 ims systemd[1]: Starting LSB: Start the Kamailio ICSCF SIP proxy server...
Apr  1 19:45:30 ims /usr/sbin/kamailio[5847]: CRITICAL: cdp [diameter_peer.c:423]: diameter_peer_destroy(): destroy_diameter_peer(): Bye Bye from C Diameter Peer test
Apr  1 19:45:31 ims icscf[7406]: Starting Kamailio ICSCF SIP Server: icscf:loading modules under config path: /usr/lib64/kamailio/modules_k/:/usr/lib64/kamailio/modules/:/usr/lib/kam
ailio/modules_k/:/usr/lib/kamailio/modules/
Apr  1 19:45:31 ims icscf[7406]: Listening on
Apr  1 19:45:31 ims icscf[7406]: udp: 127.0.0.1:6060
Apr  1 19:45:31 ims icscf[7406]: tcp: 127.0.0.1:6060
Apr  1 19:45:31 ims icscf[7406]: Aliases:
Apr  1 19:45:31 ims icscf[7406]: *: icscf.open-ims.test:*
Apr  1 19:45:31 ims icscf[7406]: *: open-ims.test:*
Apr  1 19:45:32 ims /usr/sbin/kamailio[7499]: WARNING: cdp [tcp_accept.c:121]: create_socket(): create_socket(): Trying to open/bind/listen on 127.0.0.1 port 4868
Apr  1 19:45:32 ims /usr/sbin/kamailio[7499]: WARNING: cdp [tcp_accept.c:146]: create_socket(): create_socket(): Successful socket open/bind/listen on 127.0.0.1 port 4868
Apr  1 19:45:32 ims icscf[7406]: .
Apr  1 19:45:32 ims /usr/sbin/kamailio[7498]: ERROR: cdp [peerstatemachine.c:634]: I_Snd_CER(): I_Snd_CER(): Error on finding local host address > Socket operation on non-socket
Apr  1 19:45:32 ims systemd[1]: Started LSB: Start the Kamailio ICSCF SIP proxy server.
Apr  1 19:45:43 ims /usr/sbin/kamailio[5664]: ERROR: <script>: REGISTER (sip:alice@open-ims.test (192.168.0.102:60853) to sip:alice@open-ims.test, e3225651-1232-ee8b-a00b-04145146264
6)
Apr  1 19:45:43 ims /usr/sbin/kamailio[7427]: ERROR: <core> [core/sr_module.c:1766]: get_int_fparam(): Unexpected param type: 1280329042
Apr  1 19:45:43 ims /usr/sbin/kamailio[7427]: ERROR: ims_icscf [registration.c:78]: I_perform_user_authorization_request(): no int value param passed
Apr  1 19:45:47 ims /usr/sbin/kamailio[7430]: ERROR: <core> [core/sr_module.c:1766]: get_int_fparam(): Unexpected param type: 1280329042
Apr  1 19:45:47 ims /usr/sbin/kamailio[7430]: ERROR: ims_icscf [registration.c:78]: I_perform_user_authorization_request(): no int value param passed
Apr  1 19:45:48 ims rtpengine[6441]: WARNING: Received invalid data on NG port (no cookie) from 127.0.0.1:33168:
Apr  1 19:45:51 ims /usr/sbin/kamailio[7431]: ERROR: <core> [core/sr_module.c:1766]: get_int_fparam(): Unexpected param type: 1280329042
Apr  1 19:45:51 ims /usr/sbin/kamailio[7431]: ERROR: ims_icscf [registration.c:78]: I_perform_user_authorization_request(): no int value param passed

```
(paste your log messages here)
```

#### SIP Traffic

```
(paste your sip traffic here)
```

### Additional Information

  * **Kamailio Version** - output of `kamailio -v`

```
version: kamailio 5.1.0-dev2 (x86_64/linux)
flags: STATS: Off, USE_TCP, USE_TLS, USE_SCTP, TLS_HOOKS, DISABLE_NAGLE, USE_MCAST, DNS_IP_HACK, SHM_MEM, SHM_MMAP, PKG_MALLOC, Q_MALLOC, F_MALLOC, TLSF_MALLOC, DBG_SR_MEMORY, USE_FUTEX, FAST_LOCK-ADAPTIVE_WAIT, USE_DNS_CACHE, USE_DNS_FAILOVER, USE_NAPTR, USE_DST_BLACKLIST, HAVE_RESOLV_RES
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535, DEFAULT PKG_SIZE 8MB
poll method support: poll, epoll_lt, epoll_et, sigio_rt, select.
id: unknown
compiled with gcc 4.7.2
```

* **Operating System**:

```
Debian 8
```

**Comment by miconda** [comment]
Maybe @ngvoice, @richardgood or @jaybeepee can review this report.

**Comment by ngvoice** [comment]
I'm pretty sure that this was fixed with commit 987c8101504cc7939c6782df55d7073f492e12c8 , as I experienced the same issue.

**Comment by miconda** [comment]
Closing, if 987c810 doesn't solve it, then reopen.
